src/fee-accounts/components/notifications/fee-account-error-notification.component.ts

```diff
@@ -0,0 +1,14 @@
+import {Component, OnInit, Input} from '@angular/core';
+@Component({
+  selector: 'app-fee-account-notification-banner',
+  templateUrl: './fee-account-error-notification.component.html',
+})
+
+export class FeeAccountErrorNotificationComponent implements OnInit {
+  constructor() {}
+@Input() errorMessages: Array<string>;
```

Comment:
Can't remember where, but read it was more efficient to use `string[]` as array type

Reply:
Done